### DISCUSSION TITLE
临时打包测试20210614

### DISCUSSION THREAD

**Discussion by chenggongchina** [jynew/jynew#180] [Announcements]
添加了windows64和android一键打包功能，这包（win64和android）都是使用一键打包工具生成。

<hr /><em>This discussion was created from the release <a href='https://github.com/jynew/jynew/releases/tag/v0.1.1-inner'>临时打包测试20210614</a>.</em>

**Comment by zefftan** [comment]
请问下载了该如何生成项目 ？
